### PR TITLE
Add Event Class 

### PR DESCRIPTION
**Pull request by ReubenChay**
Updated model to include Event class
Updated UI to be able to display Event information

### CODE REVIEW

File: src/main/java/longtimenosee/logic/commands/ListEventsCommand.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    public static final String MESSAGE_SUCCESS = "Listed all events";
+
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        model.updateFilteredEventList(PREDICATE_SHOW_ALL_EVENTS);
+        return new CommandResult(MESSAGE_SUCCESS, false, false, true);
+    }
+}
```
Comment (marcusczh) [inline src/main/java/longtimenosee/logic/commands/ListEventsCommand.java:24]: Seems like you have a missing empty line here

**Review by marcusczh — COMMENTED** [review]
LGTM! Just some minor nits!

File: src/main/java/longtimenosee/model/event/Event.java [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+ * Represents an Event in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+
+public class Event {
+
+    public static final Comparator<Event> EVENT_COMPARATOR = new Comparator<Event>() {
+        @Override
+        public int compare(Event e1, Event e2) {
+            int x = e1.getDate().getDate().compareTo(e2.getDate().getDate());
```
Comment (marcusczh) [inline src/main/java/longtimenosee/model/event/Event.java:18]: Perhaps you can use a more descriptive variable name here

Comment (ReubenChay) [reply src/main/java/longtimenosee/model/event/Event.java:18]: thank you for pointing that out!

**Pull request merged**